Rakefile

```
@@ -324,6 +326,14 @@ namespace :spec do
         t.rspec_opts = args.to_a.join(' ')
       end
     end
+
+    desc '' # "Explicitly hiding from `rake -T`"
+    RSpec::Core::RakeTask.new(:integration_autoinstrument) do |t, args|
+      t.pattern = 'spec/datadog/appsec/contrib/integration_autoinstrument/**/*_spec.rb'
```

It feels like instead it should be 2 adjustments and nested folders, because it looks like band-aid

My suggestion instead is:

```suggestion
    RSpec::Core::RakeTask.new(:integration_auto_instrument) do |t, args|
      t.pattern = 'spec/datadog/appsec/contrib/integration/auto_instrument/**/*_spec.rb'
```

and an exclude to the integration like

```ruby
    RSpec::Core::RakeTask.new(:integration) do |t, args|
      t.pattern = 'spec/datadog/appsec/contrib/integration/**/*_spec.rb'
      t.exclude_pattern = 'spec/datadog/appsec/contrib/integration/auto_instrument/**/*_spec.rb,'

```

_P.S in the rest of the code we use underscore to name `auto_instrumentation`, I think it makes sense to follow._